app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,75 @@
+class RecipesController < ApplicationController
+  before_action :authenticate_user!, except: [:public]
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  def public
```

Comment:
A user should be able to see all public recipes created by other users as well. I am able to see only public recipes that are created by the currently logged-in user on the recipes index page.

Reply:
according to the project requirements `a list of recipes created by the logged-in user was displayed as in the wireframe.`. the recipes page should only list the recipes created by the logged-in user.